auto-notify.plugin.zsh

```diff
@@ -56,7 +56,7 @@ function _auto_notify_message() {
             urgency="critical"
             transient=""
         fi
-        notify-send "$title" "$body" --app-name=zsh "$transient" "--urgency=$urgency" "--expire-time=$AUTO_NOTIFY_EXPIRE_TIME"
+        notify-send "$title" "$body" --app-name=zsh $transient "--urgency=$urgency" "--expire-time=$AUTO_NOTIFY_EXPIRE_TIME"
```

Comment:
I'm struggling to understand how this would happen 🤔 but happy to merge this if it is giving you issues

Reply:
I was able to consistently reproduce the issue. My guess is that it's actually passing in `""` into the command which is what's causing the problem. Removing the quotes fixes it.